<launch>

	<!-- launch find_object_3d launch file -->
	<include file="$(find ebot_perception)/launch/find_object_3d.launch" />
	
	<!-- run pick_and_place node -->
	<node name="pick_and_place" pkg="ebot_perception" type="arm_manipulation.py" output="screen" />

	<!-- run perception_pipeline node -->
	<node name="perception_pipeline" pkg="ebot_perception" type="perception_pipeline.py" output="screen"/>

</launch>
